python/ray/rllib/evaluation/rollout_worker.py

```diff
@@ -309,6 +312,9 @@ def make_env(vector_index):
                         config=tf.ConfigProto(
                             gpu_options=tf.GPUOptions(allow_growth=True)))
                 with self.tf_sess.as_default():
+                    if seed is not None:
+                        # ensure each worker has unique exploration behavior
+                        tf.set_random_seed(seed)
```

Comment:
Should this also call random.seed and numpy.random.seed?

Then it should be possible to write a unit test that checks two training calls with the same seed produce identical results.